### DISCUSSION TITLE
How to know the GPTK version installed + how to make OW2 working ?

### DISCUSSION THREAD

**Discussion by enzoaicardi** [Whisky-App/Whisky#551] [Q&A]
Hello world,
Does anyone know how to find out the version of GPTK used in the latest version of Whisky ?
The command `cd /usr/local/Cellar/game-porting-toolkit/ && ls` can't find any matching folder.

Also I try to start Overwatch2 from the battle.net launcher (it installed without any problem) but with or without DXVK the game crashes before it can start.

I have heard and seen tutorials regarding Crossover and CXpatcher, and I would like to understand why it is not possible to run the game with Whisky ?

Thanks in advance.

**Comment by Gnocchie** [comment]
I was able to run it on Crossover with CXpatcher and it worked perfectly.

Now trying it on Whisky i get the same problem as you, crashing when opening. Can't seem to find a solution for it yet

**Comment by krzdabrowski** [comment]
> Does anyone know how to find out the version of GPTK used in the latest version of Whisky ?
> The command cd /usr/local/Cellar/game-porting-toolkit/ && ls can't find any matching folder.

I was asking the same question and I think I found the answer.

As described in [A Hitchhiker's Guide to Whisky](https://github.com/Whisky-App/Whisky/wiki/A-Hitchhiker's-Guide-to-Whisky#whats-where), location for GPTK is `~/Library/Application Support/com.isaacmarovitz.Whisky/Libraries`

Here you can find `GPTKVersion.plist` that can be opened with text editor and presented like that:
```
		<key>build</key>
		<string>2</string>
		<key>major</key>
		<integer>1</integer>
		<key>minor</key>
		<integer>1</integer>
		<key>patch</key>
		<integer>0</integer>
```

App version is normally formatted like `major.minor.patch.build` which means I use `1.1.0.2` version of GPTK.

Hope this helps!
